### PR TITLE
NETOBSERV-699 redundant charts: hide some by default

### PR DESCRIPTION
**Pull request by jotak**
Also:
- Disambiguate some charts in the manage panel by indicating their types
- Rename some titles to be more explicit on what they are
- Remove the "localhost" switch for feature gate (?)

### CODE REVIEW

**Comment by jotak** [comment]
image

File: web/src/utils/features-gate.ts [hunk 1]
```diff
@@ -10,5 +9,5 @@ export enum Feature {
  */
 export const isAllowed = (f: Feature) => {
   const url = window.location.toString();
-  return url.includes('localhost:9000') || url.includes(`${f}=preview`);
+  return url.includes(`${f}=preview`);
```
Comment (jotak) [inline web/src/utils/features-gate.ts:12]: @jpinsonneau , as said yesterday I don't think we should have this localhost check for feature gates. As developers we don't necessarily want to get the in-progress stuff going around here and there, I think it's better to keep seeing something close to what users are seeing, and have feature gates still opt-in.

**Review by jpinsonneau — APPROVED** [review]
lgtm, thanks @jotak

**Comment by jotak** [comment]
thanks @Amoghrd @jpinsonneau 
/approve

**Pull request merged**